bigquery/google/cloud/bigquery/__init__.py

```diff
@@ -23,6 +23,9 @@
 """
 
 
+from pkg_resources import get_distribution
+__version__ = get_distribution('google-cloud-bigquery')
```

Comment:
I feel like the package name should be available as a constant somewhere already right?